agent/test-agent-cli/design/RUNBOOK.md

```diff
@@ -0,0 +1,82 @@
+# Steps to use Test-agent Command Line Interface
+`test-agent-cli` lets you write Testsys test using Bash and help in receiving and sending information from/to the Testsys cluster.
+
+## Prerequisites:
+* kind: https://kind.sigs.k8s.io/docs/user/quick-start/
+* Kubectl: https://kubernetes.io/docs/tasks/tools/
+* Docker : https://docs.docker.com/get-started/
+
+## Steps to install Testsys
+1. Set the TESTSYS_DIR variable to point to the directory in which you have cloned the project. For example:`export TESTSYS_DIR="${HOME}/repos/bottlerocket-test-system"`
+2. Set alias `alias testsys="${TESTSYS_DIR}/.cargo/bin/testsys"`
+3. Install the testsys command line tool into the local CARGO_HOME as: 
+```shell
+cd "${TESTSYS_DIR}"
+cargo install --path "${TESTSYS_DIR}/bottlerocket/testsys" --force
+```
+
+## Steps to create Bash based TestAgent:
+
+1. Create a Bash Test using the following commands to achieve the desired behavior of test.
```

Comment:
The following commands can be used to communicate with a testsys cluster. Create a bash script like ...